app/templates/components/ember-survey.hbs

```diff
@@ -0,0 +1,7 @@
+{{#if config.link}}
+  <div class="survey-reminder-wrapper">
+    <div class="survey-reminder-container">
+      Take the <a href={{config.link}} target="_blank" rel="noopener">{{config.title}}</a>
```

Comment:
This can actually be a simple link-to now that it is linking inside the current application 🎉

Reply:
@mansona Any thoughts on shipping these changes as-is and I can then refactor to use an internal route/link-to when I implement #173? Doing that just means I don't need to create a route/empty template for this PR.

Reply:
@mansona I've updated the PR to:

* Add a `ember-community-survey-2019` route/template
* Updated the environment survey config to have a `route` property instead of `link`
* Refactored `ember-survey` to use `config.route` in `link-to` and not show the banner on the currently configured survey route
* Added some rendering tests for `ember-survey`'s show/hide logic